src/MOI_wrapper.jl

```diff
@@ -134,6 +134,18 @@ function _allocate_constraint(cone::ConeData, f, s::MOI.ExponentialCone)
     cone.ep += 1
     ci
 end
+constroffset(cone::ConeData, ci::CI{<:MOI.AbstractFunction, <:MOI.PowerCone}) = cone.f + cone.l + cone.q + cone.s + cone.ep + ci.value
+function _allocate_constraint(cone::ConeData, f, s::MOI.PowerCone)
+    ci = length(cone.p)
+    push!(cone.p, s.exponent)
+    ci
+end
+constroffset(cone::ConeData, ci::CI{<:MOI.AbstractFunction, <:MOI.DualPowerCone}) = cone.f + cone.l + cone.q + cone.s + cone.ep + ci.value
+function _allocate_constraint(cone::ConeData, f, s::MOI.DualPowerCone)
+    ci = length(cone.p)
+    push!(cone.p, - s.exponent)
```

Comment:
I didn't know the dual power cone is simply the power cone with opposite exponent. Could you give a reference or an argument to support the claim ?

Reply:
This is the convention for SCS. There are several mentions of it in the source code: 
https://github.com/cvxgrp/scs/blob/b03e156d06bec13c3190c47b4151d94bec4a6a55/src/cones.c#L775
https://github.com/cvxgrp/scs/blob/b03e156d06bec13c3190c47b4151d94bec4a6a55/include/scs.h#L73-75
However, I did not find anything about it in the documentation…

Reply:
Oh I see, it is not equivalent but that's how you specify it to SCS. Could you add a comment explaining that ?